applications/CoSimulationApplication/python_scripts/solver_wrappers/external/flux_wrapper.py

```diff
@@ -0,0 +1,712 @@
+import os
+import sys
+
+def Create(settings, model, solver_name):
+    """
+    Create an instance of the FluxWrapper.
+
+    Args:
+        settings (Kratos.Parameters): The Kratos parameters containing the settings for the Flux wrapper.
+        model (Kratos.Model): The Kratos model containing the problem's data.
+        solver_name (str): The name of the solver associated with this wrapper.
+
+    Returns:
+        FluxWrapper: An instance of the FluxWrapper class.
+    """
+    return FluxWrapper(settings, model, solver_name)
+
+class FluxWrapper(CoSimulationSolverWrapper):
+    """
+    This class provides a dedicated Kratos wrapper for Flux 2D and 3D solvers.
+
+    This wrapper inherits from CoSimulationSolverWrapper and defines the standard Kratos wrapper functions for Flux solvers.
+
+    Attributes:
+        settings (Kratos.Parameters): The Kratos parameters containing the settings for the Flux wrapper.
+        model (Kratos.Model): The Kratos model containing the problem's data.
+        solver_name (str): The name of the solver associated with this wrapper.
+
+    Methods:
+        __init__(self, settings, model, solver_name): Initialize the Flux wrapper instance.
+
+    Note:
+        The FluxWrapper class serves as a bridge between the Flux solver and the Kratos CoSimulation framework.
+    """
+
+    def __init__(self, settings, model, solver_name):
+        """
+        Initialize the FluxWrapper instance.
+
+        Args:
+            settings (Kratos.Parameters): The Kratos parameters containing the settings for the Flux wrapper.
+            model (Kratos.Model): The Kratos model containing the problem's data.
+            solver_name (str): The name of the solver associated with this wrapper.
+        """
+        super().__init__(settings, model, solver_name)
+
+        # Check for a subprocess call
+        world_rank = 0
+        if KM.IsDistributedRun() :
+            world_data_comm = KM.ParallelEnvironment.GetDataCommunicator("World")
+            world_rank = world_data_comm.Rank()
+
+        # Prevent any launch of a FluxServer from a subprocess
+        if world_rank != 0:
+            return
+
+        # Set default settings and validate JSON settings
+        self.settings["solver_wrapper_settings"].ValidateAndAssignDefaults(self.GetDefaultSolverWrapperParameters())
+
+        # ---------------------------
+        # Import FluxPythonServer API
+        # ---------------------------
+        # Getting the path to the FluxPythonServer API
+        flux_install_path = self.settings["solver_wrapper_settings"]["flux_install_path"].GetString()
+
+        # If not defined, importing the local version of FluxPythonServer API
+        if not flux_install_path:
+            flux_install_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "../..", "Api", "Python"))
+        sys.path.append(flux_install_path)
+
+        # Importing the FluxPythonServer API
+        import FluxPythonServer as FXapi
```

Comment:
can you remove this again after the import?

Reply:
Okay